ui/src/CurrentUser.ts

```diff
@@ -144,5 +145,6 @@ export class CurrentUser {
             () => this.tryReconnect(true),
             this.reconnectTime
         );
+        this.reconnectTime = Math.min(this.reconnectTime * 2, 120000);
```

Comment:
I think we do not need this, there is no real cost in sending the requests more frequent..

Reply:
It's just a common pattern for stuff like this but I'm open to your proposal. Considering gotify is offline for a while that'd mean 1200 req/h for nothing